fonts/font.tsx

```diff
@@ -0,0 +1,5 @@
+import { Dosis, Open_Sans } from "next/font/google";
```

Comment:
Now I see. font is not a subfolder, but the name of the file.
Instead of combining all fonts in one file, please create one for each.

@/fonts/Dosis.tsx
@/fonts/Open_Sans.tsx